docs/getting-started/prerequisites.md

```diff
@@ -35,7 +35,7 @@ Install Docker using the instructions
 Googlers can visit [go/installdocker](https://goto.google.com/installdocker).
 
 If you want to run `docker` without `sudo`, you can
-[create a docker group](https://docs.docker.com/engine/installation/linux/ubuntulinux/#/create-a-docker-group).
+[create a docker group](https://docs.docker.com/engine/install/linux-postinstall/).
```

Comment:
Let's link to https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user to be extra specific.

Reply:
Sure. Updated.